diktat-rules/src/test/resources/test/paragraph1/naming/file/fileNameTest.kt

```diff
@@ -1,12 +0,0 @@
-package org.cqfn.diktat.resources.test.paragraph1.naming.file.resources.test.paragraph1.naming.file.naming.file
```

Comment:
very strange - looks like idea decided to remove these files, I will have a look